src/lib/containers/personal_access_token/CreateAccessTokenModal.tsx

```diff
@@ -103,7 +103,7 @@ export const CreateAccessTokenModal: React.FunctionComponent<CreateAccessTokenMo
           </>
         ) : (
           <div className="SRC-marginFive">
-            <form>
+            <Form>
```

Comment:
I assume this is for styling purposes only, right?  Out of curiosity, does it look right without the form (because we really don't use any of the form functionality here)?

Reply:
Form is not necessary. Getting rid of it fixes the issue!

Reply:
Yay!